docs/moduleAna3AIMEC.rst

```diff
@@ -293,8 +304,27 @@ with the analysis results (see :py:func:`ana3AIMEC.ana3AIMEC.postProcessAIMEC` f
 In this dataFrame there are multiple columns, one for each result from the analysis
 (one column for runout length, one for MMA, MAM...) and one row for each simulation analyzed.
 
+
+Analyze with respect to reference data sets
+----------------------------------------------
+In order to include a comparison to reference data sets, the flag ``includeReference`` in your local copy of ``ana3AIMEC/ana3AIMECCfg.ini``
+needs to be set to ``True``. After reading the coordinates of the point, line or polygon from the respective shp file, a raster with the same extent
+as the DEM is created with values indicating which cells are affected by the features (see :py:func:`ana3AIMEC.ana3AIMEC.postProcessReference`).
+This raster is then transformed into the thalweg following coordinate system. From the transformed rasters, a runout line/point is computed.
+This is done by identifying for each L coordinate (across thalweg) the corresponding S coordinate (along thalweg) for
+the point, line or polygon feature) (see :py:func:`ana3AIMEC.aimecTools.computeRunoutLine`).
+In case of the polygon the S coordinate furthest in thalweg direction is chosen.
+Regarding the simulation results, also a runout line is derived. This is done by identifying for each simulation, the
+last point along the thalweg where the chosen threshold of the chosen ``runoutResType`` is still exceeded (similar to the
+identification of the runout point), for each L coordinate (across thalweg).
+For each simulation the derived runout line is compared to the runout line derived from the reference line or polygon, whereas
+in the case of the reference points just a distance between this point and the simulation runout point is computed.
+The derived difference measures are saved to the aimec result DataFrame, see :ref:`moduleAna3AIMEC:List of Aimec result variables`
+and several plots are created, see :ref:`moduleAna3AIMEC:Reference data set analysis plots`.
```

Comment:
```suggestion
To include a comparison to reference data sets, the flag ``includeReference`` in your local copy of ``ana3AIMEC/ana3AIMECCfg.ini``must be set to ``True``. After reading the coordinates of the point, line or polygon from the respective shp file, a raster with the same extent as the DEM is created with values indicating which cells are affected by the features (see py:func:`ana3AIMEC.ana3AIMEC.postProcessReference`).
This raster is then transformed into the thalweg following coordinate system. From the transformed rasters, a runout line/point is computed.
This is done by identifying for each L coordinate (across thalweg) the corresponding S coordinate (along thalweg) for
the point, line or polygon feature) (see :py:func:`ana3AIMEC.aimecTools.computeRunoutLine`).
In case of the polygon the S coordinate furthest in thalweg direction is chosen. A runout line is also derived from the simulation results. This is done by identifying, for each simulation, the
last point along the thalweg where the chosen threshold of the chosen ``runoutResType`` is still exceeded (similar to the
identification of the runout point), for each L coordinate (across thalweg).
For each simulation, the derived runout line is compared to the runout line derived from the reference line or polygon, whereas
in the case of the reference points, a distance between this point and the simulation runout point is computed.
The derived difference measures are saved to the aimec result DataFrame, see :ref:`moduleAna3AIMEC:List of Aimec result variables`
and several plots are created, see :ref:`moduleAna3AIMEC:Reference data set analysis plots`.
```

Reply:
done